htsinfer/utils.py

```diff
@@ -0,0 +1,36 @@
+"""Utilities used across multiple HTSinfer modules."""
+
+from pandas import DataFrame  # type: ignore
+
+
+def results_validator(
```

Comment:
Maybe you could find a more specific name for this. What if we need another "results validator"? Something like "validate_dataframe_column_by_min_value_and_ratio" or rather "validate_sorted_vector_by_min_value_and_ratio" (see comment below)